Azure/sonic-buildimage
# [DELL][Z9264f] Adding new SKU to Z9264f

**- What I did**
Added 40G SKU for Dell platform Z9264f. Fixed issues in platform scripts

**- How I did it**
Created a SKU under Z9264f directory

**- How to verify it**
Load default SKU as well as newly created SKU. Check if ports are created and check NPU dump commands to dump the port speed

**- Description for the changelog**

Created new SKU for Z9264f

**- A picture of a cute animal (not mandatory but encouraged)**
